src/Controllers/auth.js

```diff
@@ -0,0 +1,60 @@
+const jwt = require('jsonwebtoken'),
+      client = require('./db')
+
+function getToken(values) {
+    return jwt.sign({...values}, process.env.JWT_SECRET, { expiresIn: '1d' })
+}
+
+function verifyToken( req, res, next) {
+
+    const authHeader = req.headers['authorization']
+    const token = authHeader.split(' ')[1]
+
+    jwt.verify(token, process.env.JWT_SECRET, async (err,user) => {
+        if(err){
+            console.log(user);
+            console.log(err);
+            res.status(403).json({ message : 'You are not authorized', errors: err })
+        }
+        switch(user.role) {
```

Comment:
Transfer this switch logic to a separate function and use it as a middleware
**verifyToken** function's only task should be to verify token

Reply:
Done sir!